config/config.h.in

```diff
@@ -30,6 +30,7 @@
 #cmakedefine HAVE_NL_LANGINFO_CODESET
 #cmakedefine HAVE_NL_MSG_CAT_CNTR
 #cmakedefine HAVE_PUTENV
+#cmakedefine HAVE_PUTENV_S
```

Comment:
`HAVE_PUTENV` is not actually used anywhere, so it could be removed from here and `config/CMakeLists.txt`.